src/DssSpell.sol

```diff
@@ -42,66 +38,97 @@ contract DssSpellAction is DssAction {
-
-        // Increase ETH-A Stability Fee by 0.25% from 1.5% to 1.75%.
-        DssExecLib.setIlkStabilityFee("ETH-A", ONE_PT_SEVEN_FIVE_PCT_RATE, true);
-
-        // Increase ETH-B Stability Fee by 0.25% from 3% to 3.25%.
-        DssExecLib.setIlkStabilityFee("ETH-B", THREE_PT_TWO_FIVE_PCT_RATE, true);
-
-        // Increase ETH-C Stability Fee by 0.25% from 0.75% to 1%.
-        DssExecLib.setIlkStabilityFee("ETH-C", ONE_PCT_RATE, true);
-
-        // Increase WSTETH-A Stability Fee by 0.25% from 1.5% to 1.75%.
-        DssExecLib.setIlkStabilityFee("WSTETH-A", ONE_PT_SEVEN_FIVE_PCT_RATE, true);
-
-        // Increase WSTETH-B Stability Fee by 0.25% from 0.75% to 1%.
-        DssExecLib.setIlkStabilityFee("WSTETH-B", ONE_PCT_RATE, true);
-
-        // Increase RETH-A Stability Fee by 0.25% from 0.5% to 0.75%.
-        DssExecLib.setIlkStabilityFee("RETH-A", ZERO_PT_SEVEN_FIVE_PCT_RATE, true);
-
-        // Increase CRVV1ETHSTETH-A Stability Fee by 0.25% from 1.5% to 1.75%.
-        // NOTE: ignore in goerli because CRVV1ETHSTETH-A doesn't exist there.
-        // DssExecLib.setIlkStabilityFee("CRVV1ETHSTETH-A", ONE_PT_SEVEN_FIVE_PCT_RATE, true);
-
-
-        // Increase the WSTETH-A gap by 15 million DAI from 15 million DAI to 30 million DAI.
-        // Increase the WSTETH-A ttl by 21,600 seconds from 21,600 seconds to 43,200 seconds
-        DssExecLib.setIlkAutoLineParameters("WSTETH-A", 500 * MILLION, 30 * MILLION, 12 hours);
-
-        // Increase the WSTETH-B gap by 15 million DAI from 15 million DAI to 30 million DAI.
-        // Increase the WSTETH-B ttl by 28,800 seconds from 28,800 seconds to 57,600 seconds.
-        DssExecLib.setIlkAutoLineParameters("WSTETH-B", 500 * MILLION, 30 * MILLION, 16 hours);
-
-        // Reduce the WBTC-A gap by 10 million DAI from 20 million DAI to 10 million DAI.
-        DssExecLib.setIlkAutoLineParameters("WBTC-A", 500 * MILLION, 10 * MILLION, 24 hours);
-
-        // Reduce the WBTC-B gap by 5 million DAI from 10 million DAI to 5 million DAI.
-        DssExecLib.setIlkAutoLineParameters("WBTC-B", 250 * MILLION, 5 * MILLION, 24 hours);
-
-        // Reduce the WBTC-C gap by 10 million DAI from 20 million DAI to 10 million DAI.
-        DssExecLib.setIlkAutoLineParameters("WBTC-C", 500 * MILLION, 10 * MILLION, 24 hours);
+        // --------- Collateral Offboardings ---------
+        // Poll: https://vote.makerdao.com/polling/QmPwHhLT#poll-detail
+        // Forum: https://forum.makerdao.com/t/decentralized-collateral-scope-parameter-changes-1-april-2023/20302
+
+        // Set Liquidation Penalty (chop) to 0%.
+        DssExecLib.setIlkLiquidationPenalty("YFI-A", 0);
+        // Set Flat Kick Incentive (tip) to 0.
+        // We are using low level methods because DssExecLib only allows setting `tip < WAD`: https://github.com/makerdao/dss-exec-lib/blob/69b658f35d8618272cd139dfc18c5713caf6b96b/src/DssExecLib.sol#L767
+        DssExecLib.setValue(DssExecLib.clip("YFI-A"), "tip", 0);
```

Comment:
`setKeeperIncentiveFlatRate` would have done the same thing here, no? `tip<WAD` doesn't make it an issue where tip is 0 (as it is in this case).

Reply:
2022-11-28 does it no problem for ilks:

`DssExecLib.setKeeperIncentiveFlatRate(_ilk, 0);`

Either way your code here means the same thing, but you can in fact use the execlib function here :)

Reply:
Good point! I've changed it to `setKeeperIncentiveFlatRate` to make the spell more readable